WebInstall/explainer_cross_domain.md

```diff
@@ -10,37 +10,57 @@ This document is a starting point for engaging the community and standards bodie
 
 ## Goals
 
 * **Enable installation of web apps (cross-origin).**
-* Allow a ***vetted* installation origin** to know if the web app is installed (see *[`install_sources`](#install-sources-manifest-field) new manifest field*).
+
 * Allow the web app to report to the installation origin the outcome of the installation.
 * Enable UAs to [supress potential installation-prompt spam](#avoiding-installation-prompt-spamming).
 * Track campaign IDs for marketing campaigns (with the [Acquisition Info API](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/AcquisitionInfo/explainer.md)).
 
 ## Non-goals
 
 * Install same-origin content (see [Web Install - same-origin explainer](./explainer_same_domain.md)).
-* Install arbitrary web content that is not an app (target must have a manifest file with [`install_sources`](#install-sources-manifest-field)).
+* Install arbitrary web content that is not an app (target must have a manifest file and an id). [Reasons expanded here](https://docs.google.com/document/d/19dad0LnqdvEhK-3GmSaffSGHYLeM0kHQ_v4ZRNBFgWM/edit#heading=h.koe6r7c5fhdg).
 * Change the way the UA currently prompts for installation of a web app.
 * Associate ratings and reviews with the installed app ([see Ratings and Reviews API explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/RatingsAndReviewsPrompt/explainer.md)).
 * Process payments for installation of PWAs ([see Payment Request API](https://developer.mozilla.org/en-US/docs/Web/API/Payment_Request_API)).
 * List purchased/installed goods from a store ([see Digital Goods API](https://github.com/WICG/digital-goods/blob/main/explainer.md)).
 * Installing content that does not pass the installability criteria (see *[installability criteria](#installability-criteria)*).
+* Define what "installation" means. This is up to each platform and overall we refer to the acquisition of an app onto a device.
 
 ## Use Cases
 
-The Web Install API enables installation of cross-origin applications. A website will be able to include a button to install a related application, which can be hosted in another domain. The functionality can also be used to create online repositories of apps, improving discoverability of applications.
+The Web Install API enables installation of cross-origin web applications. A website will be able to include a button to install an application, which can be hosted in another domain. These are some examples of use cases enabled by the new API.
+
+### **Websites installing their web apps**
+
+Picture a user browsing on their favorite video streaming web app. The user might browse to this web app daily, yet not be aware that there is a way that they can get the app directly from the app's UI. This could be through a button that the webapp would be able to implement, that would trigger the installation.
```

Comment:
I'm not passionate about it, so feel free to reject the suggestion if you don't agree
```suggestion
Picture a user browsing on their favorite video streaming web app. The user might browse to this web app daily, yet not be aware that there is a way that they can install the app directly from the app's UI. This could be through a button that the webapp would be able to implement, that would trigger the installation.
```